src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -82,11 +94,15 @@ export class BiggiveMainMenu {
       throw new Error('Missing subMenuElements');
     }
 
+    console.log({ subMenuElements });
```

Comment:
I think we could tidy up by removing this and the next 2 log lines, but not sure if they are useful for Donate testing and kept intentionally for now?

Reply:
Thanks, sorry seems to be a habit for me to put these in temporarily when I'm working and forget to remove. Will try to stop doing that.